apps/web/src/pages/events/[id].tsx

```diff
@@ -3,48 +3,274 @@ import { createServerSideHelpers } from "@trpc/react-query/server"
 import { appRouter, createContextInner, transformer } from "@dotkomonline/gateway-trpc"
 import { type FC } from "react"
 import { Button } from "@dotkomonline/ui"
+import clsx from "clsx"
 import { trpc } from "@/utils/trpc"
+import { AttendanceGroup } from "./AttendanceGroup"
+import { useSession } from ".."
+
+interface StatusCardProps {
+  title: string
+  text: string
+  background: string
+}
+
+// Biome ignores do not work in the middle of jsx so this is extracted just to igonre the rule here
+//biome-ignore lint/security/noDangerouslySetInnerHtml: <We do not pass any user input into this, so it is safe>
+const p = (text: string) => <p dangerouslySetInnerHTML={{ __html: text }} />
+
+const StatusCard = ({ title, text, background }: StatusCardProps) => (
+  <div className="mb-4">
+    <div className={`block rounded-lg ${background} p-4 shadow-lg`}>
+      <p className="text-lg font-bold">{title}</p>
+      {p(text)}
+    </div>
+  </div>
+)
+
+type StatusState = "CLOSED" | "NOT_OPENED" | "OPEN"
+
+const STATUS_STATE_COLOR: { [key in StatusState]: `bg-${string}-4` } = {
+  NOT_OPENED: "bg-red-4",
+  OPEN: "bg-green-4",
+  CLOSED: "bg-purple-4",
+}
+
+const STATUS_TEXTS: { [key in StatusState]: { title: string } } = {
+  OPEN: { title: "Åpen" },
+  NOT_OPENED: { title: "Ikke åpnet" },
+  CLOSED: { title: "Stengt" },
+}
+
+interface DateString {
+  value: string
+  isRelative: boolean
+}
+
+// todo: move out of file
+const dateToString = (attendanceOpeningDate: Date): DateString => {
```

Comment:
Can we use Intl.RelativeDateFormat? Alternatively use a library like date-fns to avoid all the manual date handling?